src/components/SettingsView/SettingsView.js

```diff
@@ -70,6 +72,17 @@ const SettingsView = ({
             }
             label="Show new notifications"
           />
+          <FormControlLabel
+              control={
+                <Switch
+                    color="primary"
+                    checked={hideHomeScreen}
+                    onChange={handleHideHomeScreenChange}
+                    name="hide-home-screen"
+                />
+              }
+              label="Hide Home Screen"
+          />
```

Comment:
As mentioned in my comment in `Farmhand.js`, I feel this should use "show" semantics instead of "hide." Also, it looks like this is using inconsistent indentation with the rest of the file. Running `npx prettier 'src/components/SettingsView/SettingsView.js' --write` should be all we need!

Reply:
Darnit, sorry IntelliJ hijacks the formatting sometimes, I'd thought I'd fixed them all. Done and done!

Reply:
I think I have IntelliJ setup right now, but if anyone has a CodeStyles I can borrow to make sure, that would be awesome.

Reply:
I'm not a IntelliJ user, so I'm not familiar with CodeStyles (I'm guessing you're referring to [this](https://www.jetbrains.com/help/idea/configuring-code-style.html#create-copy)). I think you might be the first one on this project! If you believe a CodeStyles configuration would helpful for other IntelliJ users, please feel free to add one in a new PR. 

In the meantime, we do have some automation around code linting and formatting. Specifically, we have this Git hook: https://github.com/jeremyckahn/farmhand/blob/32648fc0fe3d6e499e2898cda232e0cbd2a68b49/package.json#L118-L120

However, that only runs when you're on the `develop` branch. I made that the case so frequent commits on a local topic branch wouldn't get bogged down by a heavy pre-commit hook and instead let [the CI system](https://github.com/jeremyckahn/farmhand/actions/workflows/ci.yml) do it asynchronously. So, the formatting wasn't automatically cleaned up here.

I'm realizing now that linting and formatting should happen for commits on all branches, but only the test runner part of the hook should be limited to the local `develop` branch. I can make that change in a separate PR (and also clean up a variety of unrelated formatting violations) so that these sorts of things can be automated for all going forward!

Reply:
#240 has been merged and should prevent formatting violations from getting committed going forward. Thank you for helping us find this misconfiguration @UnquietBones!